### PR TITLE
Issue #SC-1617- Druid Automation script for druid ingestion health

### PR DESCRIPTION
**Pull request by RevathiKotla**

### CODE REVIEW

File: ansible/roles/druid-ingestion-health/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+druid_path : "/data/druid"
+druid_overlord_host: 11.2.1.20
```
Comment (Kaali09) [inline ansible/roles/druid-ingestion-health/defaults/main.yml]: @RevathiKotla Please variablise the ips.

**Comment by rjshrjndrn** [comment]
@RevathiKotla awesome work dude. :+1:

File: ansible/roles/druid-ingestion-health/defaults/main.yml (hunk 1 above)
Comment (SanthoshVasabhaktula) [inline ansible/roles/druid-ingestion-health/defaults/main.yml]: Shouldn't the ip addresses be variabilized?

File: ansible/roles/druid-ingestion-health/tasks/ingestion.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+- name: Alert slack after restarting the ingestion
+  slack:
+    token: '{{ druid_slack_token }}'
+    msg: '{{ item }} ingestion restarted'
+    channel: '{{ druid_slack_channel  }}'
+    username: '{{ druid_slack_user }}'
+    color: 'good'
+    icon_emoji: ':bangbang:'
+  with_items: "{{ failed_datasources }}"
```
Comment (SanthoshVasabhaktula) [inline ansible/roles/druid-ingestion-health/tasks/ingestion.yml]: How do we know it is good? Shouldn't we wait for some time (may be 5 mins) and recheck with the status API?

File: ansible/roles/druid-ingestion-health/tasks/segments.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  uri:
+    url: "{{ segments_stats_url }}"
+    method: GET
+    return_content: yes
+  register : segment_stats
+
+- name: Check the segment percent
+  set_fact:
+    status: "Failed"
+  when: item.value < 95
```
Comment (SanthoshVasabhaktula) [inline ansible/roles/druid-ingestion-health/tasks/segments.yml]: Shouldn't it be item.value != 100?

File: ansible/roles/druid-ingestion-health/tasks/segments.yml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+- name: Alert slack once the historical is up
+  slack:
+    token: '{{ druid_slack_token }}'
+    msg: 'Back to running'
+    color: 'good'
+    channel: '{{ druid_slack_channel  }}'
+    username: '{{ druid_slack_user }}'
+    icon_emoji: ':bangbang:'
+  when: status is defined and status  == "Failed"
+
```
Comment (SanthoshVasabhaktula) [inline ansible/roles/druid-ingestion-health/tasks/segments.yml]: It should check for segments status after a fixed duration? Or the alert should just say restarted and will check again in X minutes

File: ansible/roles/druid-ingestion-health/defaults/main.yml (hunk 1 above)
Comment (RevathiKotla) [reply ansible/roles/druid-ingestion-health/defaults/main.yml]: variablized the ip address

**Pull request merged**